python/lsst/cp/verify/verifyStats.py

```diff
@@ -330,6 +346,9 @@ def run(
         else:
             outputStats["DET"] = {}
 
+        if self.config.useIsrStatistics:
+            outputStats["EXTRA"] = isrStatistics
```

Comment:
Why is the key called `EXTRA` instead of something more specific?

Reply:
Lack of imagination and a healthy fear of #dm-naming-things.  I've switched this to `ISR`, as these are the quantities measured during that stage.